src/adapter.ts

```diff
@@ -70,6 +82,26 @@ class Adapter {
     return this.ready;
   }
 
+  getManager() {
+    return this.manager;
+  }
+
+  isVerbose() {
+    return this.verbose;
+  }
+
+  getGatewayVersion() {
+    return this.gatewayVersion;
+  }
+
+  getUserProfile() {
+    return this.userProfile;
+  }
+
+  getPreferences() {
+    return this.preferences;
+  }
+
   asDict() {
```

Comment:
Any chance to add return type here?

Reply:
Sure!

Reply:
There are more methods without return type, so why not all? :-D

Reply:
I added return types for almost all of them after adding `@typescript-eslint/recommended`.